AE/README.md

```diff
@@ -0,0 +1,44 @@
+# RECom Artifact Evaluation
+
+The artifact contains the necessary software components to validate the main results in the RECom paper.
```

Comment:
It would be better to split the following content into two subsections: # Run with a single command and # Run step by step.